src/js/functions.js

```diff
@@ -0,0 +1,94 @@
+export const createCard = user => {
+  const card = document.createElement("section");
+  card.classList = "swarm__main__container__item";
+  ////
```

Comment:
Not sure what this comments mean

Reply:
We wanted to visual separate some steps. We may should have type a little comment, so extern viewers can understand this separation more easily. Thx for the feedback, by the way!

Reply:
You're right, this comments are very cryptic for people outside the harmony-team and  make no sense in a repository.

Reply:
I'm refactoring the functions.js file and delete this comments.